scripts/recovery_sync

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+function die
+{
+    echo "$1" >&2
+    exit 1
+}
+
+[[ $# -eq 1 ]] || die "Illegal number of parameters"
+target="$1"
+workdir=$(mktemp -d)
+
+cd $workdir
+sudo chown root .
+sudo chmod 0755 .
+cat $target | gunzip | sudo cpio -idmv
+
+sudo rsync -a /run/systemd/network/ etc/systemd/network
+sudo mkdir -p etc/dropbear
+for fmt in $(ls /etc/ssh/*key | sed 's/.*host_\(.*\)_key/\1/'); do
```

Comment:
Are we copying configuration from the build server here?

Reply:
No, recovery_sync is run on the engine when the networking or other config is changed.

Reply:
Ah, ok.. I must not have looked closely enough.. I see now the Makefile is just copying it into place, rather than executing it. Sorry for the confusion.